package-lock.json

```diff
@@ -570,6 +570,16 @@
             }
           }
         },
+        "raw-loader": {
```

Comment:
Pls revert this, new version fo row loader breaks everything

Reply:
Ok, will do
But all I did is just `npm install`, should I use `npm ci` instead?

Reply:
@aintnomoreloyalty yea, just use `npm ci`